modules/tide_oauth/src/EnvKeyGenerator.php

```diff
@@ -89,7 +100,7 @@ public function hasEnvKeys() : bool {
    */
   public function generateEnvKeys() : bool {
     if (!$this->hasEnvKeys()) {
-      return FALSE;
+      $this->generateOauthKeys();
```

Comment:
@MdNadimHossain should it return right here? `$this->generateOauthKeys()` generates the keypair, and the subsequent lines overwrite the keypair with values of the env variables -  which do not exist.

Reply:
@sonnykt oh yes it shouldn't return. It should just create the oauth.key and oauth.pub if  EnvKeys doesn't exist. Then it will continue to create the private and public key right!

Reply:
@sonnykt Updated thanks